VideoLocker/src/main/java/org/edx/mobile/view/CourseDetailFragment.java

```diff
@@ -0,0 +1,268 @@
+        courseAbout = (LinearLayout) view.findViewById(R.id.course_detail_course_about);
+        courseAboutWebView = (EdxWebView) courseAbout.findViewById(R.id.course_detail_course_about_webview);
+        courseAbout.setVisibility(View.INVISIBLE);
+    }
+
+    /**
+     * Populates the information for course details.
+     */
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        setCourseImage();
+        setCourseVideoButton();
+        setCourseCardText();
+        shortDescription.setText(courseDetail.short_description);
+        setAboutThisCourse();
+    }
+
+    private void setCourseCardText() {
+        String formattedDate = CourseCardUtils.getFormattedDate(
+                getActivity(),
+                courseDetail.start,
+                courseDetail.end,
+                courseDetail.start_type,
+                courseDetail.start_display);
+        courseTextDetails.setText(CourseCardUtils.getDescription(courseDetail.org, courseDetail.number, formattedDate));
+        courseTextName.setText(courseDetail.name);
+    }
+
+    private void setCourseImage() {
+        final String headerImageUrl = environment.getConfig().getApiHostURL() + courseDetail.media.course_image.uri;
+        Glide.with(CourseDetailFragment.this)
+                .load(headerImageUrl)
+                .placeholder(R.drawable.edx_map_login)
+                .transform(new TopAnchorFillWidthTransformation(getActivity()))
+                .into(headerImageView);
+    }
+
+    private void setCourseVideoButton() {
+        if (courseDetail.media.course_video.uri == null || courseDetail.media.course_video.uri.isEmpty()) {
+            headerPlayIcon.setEnabled(false);
+        } else {
+            headerPlayIcon.setVisibility(headerPlayIcon.VISIBLE);
+        }
+    }
+
+    /**
+     * Makes a call the the course details api and sets the overview if given.
+     */
+    private void setAboutThisCourse() {
+        getCourseDetailTask = new GetCourseDetailTask(getActivity(), courseDetail.course_id) {
+            @Override
+            protected void onSuccess(CourseDetail courseDetail) throws Exception {
+                super.onSuccess(courseDetail);
+                if (courseDetail.overview != null && !courseDetail.overview.isEmpty()) {
+                    setAboutThisCourse(courseDetail.overview);
+                }
+            }
+
+            @Override
+            protected void onException(Exception e) throws RuntimeException {
+                super.onException(e);
+                showErrorMessage(e);
+            }
+        };
+        getCourseDetailTask.execute();
+    }
+
+    /**
+     * Takes a string which can include html and then renders it into the courseAbout webview.
+     *
+     * @param overview A string that can contain html tags
+     */
+    private void setAboutThisCourse(String overview) {
+        addDivider();
```

Comment:
nit: Since this divider's existence depends on the visibility of `courseAbout`, you could define it in XML as the first child of the `courseAbout` LinearLayout.

(I prefer XML layouts over code whenever reasonable)